src/test/MessageProvider.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { RawIntlProvider } from 'react-intl';
+import createIntl from './createIntl';
+
+interface Props {
+  locale?: string;
+  messages?: Record<string, string>;
+}
+
+// tslint:disable-next-line:variable-name
+const MessageProvider = ({children, ...props}: React.PropsWithChildren<Props>) => {
+  const intlObject = createIntl(props.locale, props.messages);
+
+  return <RawIntlProvider value={intlObject}>
+    {children}
+  </RawIntlProvider>;
+};
+
+export default MessageProvider;
```

Comment:
@TomWoodward I removed the polyfill stuff from this test component since I use the real MessageProvider when testing that